database/schema.sql

```diff
@@ -842,3 +842,47 @@ ALTER TABLE dbo.sccv_person_case_status_answers
   
 ALTER TABLE dbo.sccv_person_case_status_answers
 	ADD COLUMN end_date timestamp;
+
+  --MASH tables
+  CREATE TABLE IF NOT EXISTS dbo.REF_MASH_REFERRALS
+(
+  id bigint GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+  referrer varchar(256),
+  requested_support varchar(256),	
+  worker_id bigint,
+  created_at timestamp without time zone,

```

Comment:
Are you sure we don't care about timezone? Do we need to know the timezone because of the clocks going forward and back one hour for BST/UTC?

Reply:
as there is no compelling reason for excluding the timezone, will keep as regular timestamp.